scripts/compile-triton.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 set +o xtrace
-if [ ! -d "$BASE" ]; then
+if [ ! -z "$BASE" ]; then
```

Comment:
Bash `-z` is a test for empty string.
```suggestion
if [ -z "$BASE" ]; then
```

Reply:
opps